### DISCUSSION TITLE
Invalidate state not working when bind to ngModel

### DISCUSSION THREAD

**Discussion by fadelc99** [primefaces/community#1397] [PrimeNG Templates] [answered]
Hi 
        I  use prime verona 17.0.0 and found problem about invalid state not working when bind to ngModel like this picture :

<img width="1851" alt="Screenshot 2567-03-10 at 17 27 46" src="https://github.com/primefaces/community/assets/1435631/b1db9fc4-1205-46c9-b117-2900453c70f6">

How to fixed it , Thank you

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

In the examples, we're appending `ng-invalid` and `ng-dirty` classes manually to show how an invalid state looks like. As you see, we're using `class` attribute of HTML, not `ngClass`. Because the `class` overrides the `ngClass` value. That's why your classes don't appear in DOM, because you're appending classes with ngClass. `ng-invalid` and `ng-dirty` classes belong to Angular and usually it's not used like this, they are appended automatically by angular using ngClass. That's why it doesn't work. If you create a formControl and validation logic you'll see that the classes will append automatically.
